config.json

```diff
@@ -1123,6 +1123,18 @@
       "difficulty": 0,
       "topics": null,
       "deprecated": true
+    },
+    {
+      "slug": "high-scores",
```

Comment:
I know that we currently have no exercise ordering check in the tests right now because https://github.com/exercism/rust/pull/804#issuecomment-470197811 . But maybe it makes sense to move high-scores above the deprecated exercise `hexadecimal`?

This change is optional - feel free to merge without having made this change without seeking approval from me on whether it is OK to not make the change

Reply:
Good point! I hadn't actually looked at where in the configuration file this had been added. 

I agree, this exercise should fall in the region after the exercise which unlocks it (which should not be null) and before the subsequent core exercise.